src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,58 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+import { Tab } from '../../types/Tab';
+
+type Props = {
+  tabs: Tab[];
+  selectedTab: Tab;
+  selectedTabId: string;
+  onTabSelected: (tab: Tab) => void;
+};
+
+export const Tabs: React.FC<Props> = ({
+  tabs,
+  selectedTab,
+  selectedTabId,
+  onTabSelected,
+}) => {
+  let tabId = selectedTabId;
+
+  if (!tabs.find((tab) => tab.id === selectedTabId)) {
+    tabId = 'tab-1';
+  }
```

Comment:
Useless logic in Tabs component. Component should show tabs and handle tab change
```suggestion

```